docs/docs/troubleshooting.md

```diff
@@ -26,28 +26,35 @@ Variations of the warnings shown below occur when flashing the `<firmware>.uf2`
 An error along the lines of `CMake Error at (zmk directory)/zephyr/cmake/generic_toolchain.cmake:64 (include): include could not find load file:` during firmware compilation indicates that the Zephyr Environment Variables are not properly defined.
 For more information, click [here](../docs/development/setup.md#environment-variables).
 
-### dtlib.DTError
+### West build errors
 
-An error along the lines of `dtlib.DTError: <board>.dts.pre.tmp:<line number>` during firmware compilation indicates an issue within the `<shield>.keymap` file.
-This can be verified by checking the file in question, found in `mkdir/app/build`.
-
-|                  ![Example Error Screen](../docs/assets/troubleshooting/keymaps/errorscreen.png)                   |
-| :----------------------------------------------------------------------------------------------------------------: |
-| An example of the dtlib.DTError when compiling an iris with the nice!nano while the keymap is not properly defined |
-
-After opening the `<board>.dts.pre.tmp:<line number>` and scrolling down to the referenced line, one can locate errors within their shield's keymap by checking if the referenced keycodes were properly converted into the correct [USB HID Usage ID](https://www.usb.org/document-library/hid-usage-tables-12).
+West build errors usually indicates syntax problems in the `<shield>.keymap` file in the compilation process. The following are some examples and root causes.
```

Comment:
I suggest `<keyboard>.keymap` instead of `<shield>.keymap` as this applies to _boards_ as well as _shields_:

```suggestion
West build errors usually indicate syntax problems in the `<keyboard>.keymap` file during the compilation process. The following are some examples and root causes.
```

(Couple of other very minor suggestions there too…)